### PR TITLE
feat: add rsk3js tests

### PR DESCRIPTION
**Pull request by bguiz**
## What

- Same set of tests for rsk3.js as for web3.js

## Why

- verify compatibility

## Refs

- [Task](https://trello.com/c/ruv3rg8A/301)

### CODE REVIEW

**Comment by bguiz** [comment]
**do NOT merge** - https://github.com/rsksmart/rsk3.js/pull/52 needs to merged, and the resulting new version published, and this project updated to use it

**Comment by bguiz** [comment]
This is ready for review/ merge now - all 18/18 tests for rsk3.js passing against a Regtest instance running locally.

`npm run test`'s run script modified to run both files now, so the existing CI workflow should pick up on the results automatically - expect to see 36 tests now, instead of the previous 18, in the report.

File: smoke-test-web3js.js [hunk 1]
```diff
@@ -17,7 +16,7 @@ describe('RskJ Smoke Tests', function () {
 
 
   before(async () => {
-    web3 = new Web3('http://127.0.0.1:4444', null, { transactionConfirmationBlocks: 1 });
+    web3 = new Web3('http://127.0.0.1:4444/1.3.0/', null, { transactionConfirmationBlocks: 1 });
```
Comment (jonathansmirnoff) [inline smoke-test-web3js.js:19]: do we need this?

Comment (bguiz) [reply smoke-test-web3js.js:19]: Hmm I suppose not, will revert

**Comment by bguiz** [comment]
@jonathansmirnoff I've removed the version in the endpoint path

**Pull request merged**